doc-content/jbpm-docs/src/main/asciidoc/Persistence/variables-auditing-con.adoc

```diff
@@ -0,0 +1,156 @@
+[id='variables-auditing-con_{context}']
+= Auditing variables
```

Comment:
Heading needs to be "Variable auditing" (or something) since this is a con and not a proc. Need to reserve "Doing ..." titles for procs.